addons/account_edi/wizard/account_resequence.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+from odoo import _, models
+from odoo.exceptions import UserError
+
+class ReSequenceWizard(models.TransientModel):
+    _inherit = 'account.resequence.wizard'
+
+    def resequence(self):
+        edi_is_send = self.move_ids.edi_document_ids.filtered(lambda d: d.edi_format_id._needs_web_services() and d.state in ('sent'))
```

Comment:
`d.state in ('sent')`
You're only checking one state, so I guess an equality would more appropriate than an `in` ?